lib/ansible/modules/network/quagga/quagga_command.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# {c} 2017, Aleksey Gavrilov <le9i0nx+ansible@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: quagga_command
+version_added: 2.4
+short_description: Provides configuring quaggas
+description:
+    - The M(quagga_command) module takes the command name followed by a list of space-delimited arguments.
+    - The given command will be executed on all selected nodes.
+    - Documentation for M (quagga) can be found U(http://www.nongnu.org/quagga/docs.html)
```

Comment:
`M(...)` will generate a link to a module page, which in this case doesn't exist, so I think this should be

`Documentation for C(quagga) can be found U(http://www.nongnu.org/quagga/docs.html)`

http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#formatting-options